### PR TITLE
[TECH] Layout

### PR DESCRIPTION
**Pull request by kseniyaLine**

### CODE REVIEW

File: src/components/layout/Layout.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import "./Layout.module.css";
+
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: empty string

File: src/components/layout/Layout.scss [hunk 2]
```diff
@@ -4,3 +4,10 @@
   padding: 0;
   margin: 0 auto;
 }
+
+@media (max-width: 800px) {
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.scss]: so from 1280 to 800 we will have content without any borders. You can do it higher without media queries

Comment (kseniyaLine) [reply src/components/layout/Layout.scss]: @mtalstykh-exadel 
so, as i understood correctly, i must delete  everything after @media and add padding:20px in .main? i tried to make @media without width, but there were errors.

File: src/components/layout/Layout.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import "./Layout.module.css";
+import "./Layout.scss";
 import React from "react";
-import {PropTypes} from "@material-ui/core";
+import {PropTypes} from "prop-types";
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: make it { PropTypes } (with spaces)

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by tensenrey — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Pull request merged**